iDiff/differs/differs.go

```diff
@@ -13,20 +16,17 @@ var diffs = map[string]func(string, string) (string, error){
 	"apt":  AptDiff,
 }
 
-func Diff(arg1, arg2, differ string) (string, error) {
+func Diff(arg1, arg2, differ, output string) (string, error) {
 	if f, exists := diffs[differ]; exists {
 		if differ == "hist" {
 			return f(arg1, arg2)
```

Comment:
Good point - hist does not output to file at the moment.  I'm going to rework hist and dir to return JSONs when the flag is set, and similarly rework the apt differ so with the flag JSONs are returned, otherwise human-readable output is returned.  If the user wants to save the JSON to a file I think we should allow them to do that with the command line on their own.